assets/src/edit-story/app/story/useStoryReducer/reducers/selectElement.js

```diff
@@ -37,6 +37,14 @@ function selectElement(state, { elementId }) {
     return state;
   }
 
+  // If bg element was already the (only) selection, set selection to new element only
+  if (state.selection.includes(currentPage.backgroundElementId)) {
```

Comment:
Nit: `state.selection.includes` doesn't read as "the only". Are you concerned that other elements would somehow creep in?

Reply:
That background element either has to be the only selected element or not selected at all is an invariant of the system (that this specific line of code in fact maintains), so it should not need validation as it should now be impossible to get to that situation.